### DISCUSSION TITLE
go-jsonrpc@v0.1.4-0.20210217175800-45ea43ac2bec/websocket.go:122 handle me:write tcp 192.168.2.4:3456->192.168.2.100:58696: i/o timeout

### DISCUSSION THREAD

**Discussion by lilin-lab** [filecoin-project/lotus#7948] [Storage Provider]
> Note: For security-related bugs/issues, please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).

**Describe the bug**
The worker process is still there but is not doing the task

**Version (run `lotus version`):**
 go-jsonrpc@v0.1.4-0.20210217175800-45ea43ac2bec

**To Reproduce**
1. Run 'lotus-worker'
2. See error

**Expected behavior**

**Logs**
 go-jsonrpc@v0.1.4-0.20210217175800-45ea43ac2bec/websocket.go:122 handle me:write tcp 192.168.2.4:3456->192.168.2.100:58696: i/o timeout 
**Screenshots**

**Additional context**

**Comment by Reiers** [comment]
Hi @johnli-helloworld 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by LexLuthr** [comment]
@johnli-helloworld Do you still need help with this problem? If not, then can you please tell us how you fixed it?

**Reply by LexLuthr** [reply]
I am archiving this discussion due to inactivity.
